.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
```

Comment:
what do the .DS_Store and pycache signify? I understood the .env but not the other two...

Reply:
Great question.

`.DS_Store` is a hidden file created for the macOS Finder app, which is found in every directory in Unix

`__pycache__` is a folder created by Python to store compiled byte code. This stackoverflow post explains it better: https://stackoverflow.com/questions/16869024/what-is-pycache

No harm committing either of these to our repo -- just creates some clutter.